src/index.ts

```diff
@@ -26,6 +26,13 @@ function createWindow() {
     return { action: 'deny' }
   })
 
+  win.webContents.on('will-navigate', (event, url) => {
+    if(!url.startsWith('https://app.graphite.dev')) {
+      event.preventDefault();
```

Comment:
What is the default behavior we're stopping here? (I'm guessing opening a new window?)

Reply:
prevent us from navigating to the link